test-poc/base/pom.xml

```diff
@@ -55,6 +55,7 @@
             <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
+                    <argLine>-Djava.util.concurrent.ForkJoinPool.common.threadFactory=io.quarkus.bootstrap.forkjoin.QuarkusForkJoinWorkerThreadFactory</argLine>
```

Comment:
The `argLine` is actually doing the same as the `systemPropertyVariables` is doing for a different property but in a more readable syntax. Could you please try if transforming the `argLine` to a `systemPropertyVariables` solves it for you as well?